src/components/Create/components/RewardsList/RewardsList.tsx

```diff
@@ -76,7 +76,7 @@ export const RewardsList: React.FC<RewardsListProps> &
         {!!rewards.length && (
           <>
             {sortNftsByContributionFloor(rewards).map((reward, i) => (
-              <div key={reward.id}>
+              <>
                 <RewardItem
                   reward={reward}
```

Comment:
@johnnyd-eth we actually do need the div. Or else we cant set the `key` prop and react will skitz out

Reply:
Ah yep got it